notebooks/mock-notebooks/login/User Login Flow.ipynb

```diff
@@ -0,0 +1,403 @@
+{
```

Comment:
<p>When verbose=True will the process just take longer?</p><p>Wondering if the user will remember to set verbose to FALSE without documentation. Is that a typical property the user would expect to clarify for logging in.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/6333/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p> <code>verbose=False</code>  will turn off any type of acknowledgement statements/messages for the user. e.g. when a user verbose logs in with verbose=True, they we will see acknowledgement messages of connecting to domain and with which email they are logged in.</p><pre class="ql-syntax" spellcheck="false">Connecting to https://ca.openmined.com  
Logging into Canada Domain...  Logged in 🥳!!  
Logged in as sheldon@caltech.edu  
</pre>

Reply:
<p>Gotcha ^_^</p>